file-formats/enhs/enhs.json

```diff
@@ -132,8 +134,8 @@
             "description": "Example classes of the BAdI definition",
             "type": "array",
             "items": {
-              "title": "Example Class",
-              "description": "Example class",
+              "title": "Object Name",
+              "description": "Object name with max. length 30",
```

Comment:
this information is duplicated with he maxLength attribute. can you re-phrase the description. mabye just "Object name" is also ok?!

Reply:
@marcushoepfner  do you prefer "Object name" or the old title and description ("Example Class")? For the first option, I have to adopt the interface zif_aff_types_v1 with the reusable components. This would not be a big change, but I'd like to check the other option first.

Reply:
for me it is ok to take the one with less effort